cherryontech/jupiter-jumpers
# Add Logout confirm, update styles

## Description
1) Implements a logout confirmation view, shown once the user logs out successfully. 
2) Implement the new logo on the login, registration, and logout confirmation forms.
3) Update the button styles (new primary button is blue)

## Related Issue

[SCRUM 32](https://cherryontech-jupiter-jumpers.atlassian.net/browse/SCRUM-32)

## Related Story Card

## Type of Changes

Style and feature.

## Acceptance Criteria

- [X] Given I am logged in to my account and am on any page within the platform,
- [X] When I initiate logout by selecting the designated logout option (e.g., clicking "Logout" button or confirming a logout prompt),
- [X] Then the system performs the following actions:
- [X] Terminates my user session.
- [X]  Clears any session data stored on the client-side (e.g., session cookies).
- [X] Redirects me to the login screen with confirmation that user is successfully logged out.

## Update Screenshots

### Before

### After
![Screenshot 2024-05-11 at 2 17 30 PM](https://github.com/cherryontech/jupiter-jumpers/assets/47403135/524bd16b-c7e5-4e75-9267-4524b79c97bd)

## Testing Instructions

1. Log in
2. Log out
3. Confirm you can see the confirmation window
4. Click on 'Log in" and ensure you are redirected to the login form
5. Click on 'Create an account" and ensure you are redirected to the registration form

## Learnings (Optional)

_Document any things you learned or 'gotchas' you experienced._
